app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.java

```diff
@@ -409,7 +417,12 @@ public void onExitFullScreen() {
             public void onDownloadStart(Download download) {
                 if (PackageManager.PERMISSION_GRANTED == ContextCompat.checkSelfPermission(getContext(), Manifest.permission.WRITE_EXTERNAL_STORAGE)) {
                     // We do have the permission to write to the external storage. Proceed with the download.
-                    queueDownload(download);
+                    if (download.getDestinationDirectory().equals(Environment.DIRECTORY_DOWNLOADS)) {
```

Comment:
Please add a comment explaining ^ as we spoke about!

In general, if a reviewer asks a question about why something is the way it is, it may be worth adding a comment because future developers reading your code may also have the same questions!